.chloggen/add-processor-to-contrib-build.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: cmd/otelcontribcol
```

```suggestion
component: processor/metricstarttime
```
Nit: This PR's main purpose is centered on the processor